modules/apps/headless/headless-builder/headless-builder-test/src/testFunctional/macros/APIBuilderUI.macro

```diff
@@ -72,14 +72,26 @@ definition {
 	}
 
```

Comment:
I found the new poshi Signature Syntax will add 'space' between one parameter and the next one